README.md

```diff
@@ -305,6 +697,24 @@ defmodule Cookbook.AddIndexOnIngredients do
 end
 ```
 
+**GOOD ✅**
+
+Instead, start an index with columns that narrow down the results the most.
+
+```elixir
+defmodule Cookbook.AddIndexOnIngredients do
```

Comment:
I think it's better to have consistent topic across all examples (either cooking or blog posts).
I originally chose culinary topic, because I thought it's fun. 
It could be done in another PR.